apps/calendar/js/views/pan.js

```diff
@@ -188,8 +188,9 @@ Pan.prototype = {
 
   _set: function(x) {
     x = clamp(x, this._minX, 0);
+    var _x = (document.documentElement.dir === 'rtl') ? -x : x;
```

Comment:
I'm going to provide a patch soon for the `pan.js`, so don't worry about that for now.